### PR TITLE
chore: cleanup comments & READMEs

### PR DESCRIPTION
**Pull request by layershifter**
Fixing typos, references to Fluent UI, wrong naming.

### CODE REVIEW

File: apps/website/docs/react/guides/ssr-usage.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Griffel provides first class support for Server-Side Rendering.
 For basic instructions to setup Next.js, see [Getting Started](https://nextjs.org/docs/getting-started). Please complete following steps:
 
 1. Get a basic Next.js setup running, rendering a page from the `pages` folder, as guided by the tutorial.
-2. Add the Griffel to dependencies (`@griffel/react` package`), check [Install](/react/install) page.
+2. Add the Griffel to dependencies (`@griffel/react` package), check [Install](/react/install) page.
```
Comment (layershifter) [inline apps/website/docs/react/guides/ssr-usage.md:16]: Oops

File: apps/website/docs/react/guides/child-window-rendering.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ When rendering in the main browser window, many components will need access to w
 
 ## Configure rendering
 
-We need to configure a renderer for makeStyles() and pass a targetDocument to FluentProvider:
+We need to configure a renderer for `makeStyles()` and pass a `targetDocument` to `FluentProvider`:
```
Comment (ling1726) [inline apps/website/docs/react/guides/child-window-rendering.md:12]:
```suggestion
We need to configure a renderer for `makeStyles()` and pass a `targetDocument` to `RendererProvider`:
```

File: packages/babel-preset/src/index.ts [hunk 3]
```diff
@@ -8,7 +8,7 @@ export { configSchema } from './schema';
 export type { Evaluator, EvalRule } from '@linaria/babel-preset';
 export type { BabelPluginOptions };
 
-export default function linaria(babel: ConfigAPI, options: BabelPluginOptions) {
+export default function griffelPreset(babel: ConfigAPI, options: BabelPluginOptions) {
```
[the remark this answers is not in the document]
Comment (ling1726) [reply packages/babel-preset/src/index.ts:11]: this is probably the best typo so far

**Review by ling1726 — APPROVED** [review]

File: apps/website/docs/react/guides/child-window-rendering.md (hunk 2 above)
Comment (layershifter) [reply apps/website/docs/react/guides/child-window-rendering.md:12]: OMG 🤦

**Pull request merged**